src/status_im/contexts/profile/settings/events.cljs

```diff
@@ -12,7 +12,7 @@
   [{:keys [db]}]
   (let [{:keys [name preferred-name display-name]} (:profile/profile db)]
     {:json-rpc/call [{:method     "wakuext_sendContactUpdates"
-                      :params     [(or preferred-name display-name name) ""]
+                      :params     [(or preferred-name display-name name) "" ""]
```

Comment:
Similar question:
What's the meaning of the empty string?

Reply:
Same thing, I guess `wakuext_sendContactUpdates` is actually used as set ens name? not sure about it